src/__tests__/integration/event.test.js

```
@@ -0,0 +1,482 @@
+    });
+    it("Should throw an Error if the Event capacity is not of valid Number type", () => {
+      const event = new Event({ capacity: "12340xxx" });
+      const validationError = event.validateSync();
+      expect(validationError.errors.capacity).toBeDefined();
+    });
+    it("Should pass if the Event capacity is of valid Number type", () => {
+      const event = new Event({ capacity: 10000 });
+      const validationError = event.validateSync();
+      expect(validationError.errors.capacity).not.toBeDefined();
+    });
+    it.only("Should throw an error if the Event capacity does not equal the total capacity of the ticket types", async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)